ambry-commons/src/test/java/com.github.ambry.commons/TestSSLUtils.java

```diff
@@ -229,8 +234,17 @@ public static void verifySSLConfig(SSLContext sslContext, SSLEngine sslEngine, b
 
     // SSLEngine verify
     String[] enabledProtocols = sslEngine.getEnabledProtocols();
-    Assert.assertEquals(enabledProtocols.length, 1);
-    Assert.assertEquals(enabledProtocols[0], SSL_ENABLED_PROTOCOLS);
+    if (enabledProtocols.length == 2) {
+      // Apparently the Netty OpenSslEngine has no way of disabling the SSLv2Hello protocol.
+      // This is the relevant code from ReferenceCountedOpenSslEngine.getEnabledProtocols():
+      //     // Seems like there is no way to explicit disable SSLv2Hello in openssl so it is always enabled
+      //     enabled.add(PROTOCOL_SSL_V2_HELLO);
```

Comment:
The comments here need to be formatted.

Reply:
Here the indented lines were actually a quote of another piece of source code (`ReferenceCountedOpenSslEngine`). I'll add quote marks to make it more clear